website_logo/views/website_templates.xml

```diff
@@ -1,13 +1,10 @@
-<?xml version="1.0" encoding="utf-8"?>
 <odoo>
-
-    <template id="layout_logo_show" inherit_id="website.layout_logo_show">
-        <xpath expr="//a//span" position="attributes">
-            <attribute name="class" add="hide" separator=" "/>
-        </xpath>
-        <xpath expr="//a//span" position="after">
-            <img src="/website_logo.png" alt="Website logo" title="Website logo"/>
+     <template id="add_logo_to_website" inherit_id="website.layout_logo_show" customize_show="True" name="Show Logo">
+        <xpath expr="//header//a[hasclass('navbar-brand')]" position="replace">
```

Comment:
Please never do replace and use the other technique as less invasive.

Reply:
done